### PR TITLE
Feature: playing queue

### PR DESCRIPTION
**Pull request by endway**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -1,91 +1,124 @@
...
+    switch(status) {
+        case "playing":
+            if(lastPlayingTabId) {
+                chrome.tabs.sendMessage(lastPlayingTabId, {action: 'pause'});
+            }
+            break;
+
+        case "paused":
+            if(lastPlayingTabId) {
+                chrome.tabs.sendMessage(lastPlayingTabId, {action: 'play'});
```
Comment (beshur) [inline background.js]: This is Javascript police. Show me your `.editorconfig`

Comment (endway) [inline background.js]: 🚔

**Comment by endway** [comment]
@beshur Use https://github.com/StoPlay/stoplay-ext/pull/62/files?w=1 to skip `indentation changes`

File: background.js [hunk 2]
```diff
@@ -1,91 +1,124 @@
...
+    this.remove(tabId);
+    
+    queue = DataStorage.get('queue') || [];
+    queue.push(tabId);
+
+    DataStorage.set('queue', queue);
+};
+
+Queue.remove = function (tabId) {
+    DataStorage.set('queue', DataStorage.get('queue').filter(function (item) { return item !== tabId; }));
```
Comment (endway) [inline background.js]: Immutable YO!

**Comment by beshur** [comment]
Upon start

**Pull request merged**